src/test/java/org/opensearch/security/auditlog/AuditLogTestSuite.java

```diff
@@ -1,56 +0,0 @@
-/*
```

Comment:
Lets drop this change to a separate PR to keep it focused, what do you think?

Reply:
Yeah of course. I think it just got included cause I was running on top of your branch.